src/components/toolbar/toolbarFieldExport.js

```diff
@@ -138,37 +103,35 @@ const useExport = ({
  * @param {object} options
  * @param {Function} options.useExport
  * @param {Function} options.useProduct
- * @param {Function} options.useProductInventoryQuery
+ * @param {Function} options.useProductExportQuery
  * @returns {Function}
  */
 const useOnSelect = ({
   useExport: useAliasExport = useExport,
   useProduct: useAliasProduct = useProduct,
-  useProductInventoryQuery: useAliasProductInventoryQuery = useProductInventoryHostsQuery
+  useProductExportQuery: useAliasProductExportQuery = useProductExportQuery
 } = {}) => {
-  const createExport = useAliasExport();
+  const { createExport } = useAliasExport();
   const { productId } = useAliasProduct();
-  const inventoryQuery = useAliasProductInventoryQuery();
+  const exportQuery = useAliasProductExportQuery();
 
   return ({ value = null } = {}) => {
     const sources = [
       {
         application: APP_TYPES.SUBSCRIPTIONS,
         resource: RESOURCE_TYPES.SUBSCRIPTIONS,
         filters: {
-          ...inventoryQuery,
-          productId
+          ...exportQuery
         }
       }
     ];
 
-    const data = { format: value, name: `${EXPORT_PREFIX}-${productId}`, sources };
-    createExport({ data });
+    createExport(productId, { format: value, name: `${EXPORT_PREFIX}-${productId}`, sources });
```

Comment:
required: review creating something similar with the dotenv and related filename helper for consistency